draft-ietf-quic-manageability.md

```diff
@@ -1113,7 +1113,7 @@ connection could result in variations in order, delivery rate, and drop rate.
 As feedback about loss or delay of each packet is used as input to
 the congestion controller, these variations could adversely affect performance.
 Depending on the loss recovery mechanism implemented, QUIC may be
-more tolerant of packet re-ordering than traditional TCP traffic (see
+more tolerant of packet re-ordering than TCP traffic (see
```

Comment:
While "traditional" is not an accurate wording here, we need some adjective here because TCP with RACK might be as tolerant than QUIC.